twind.install.js

```diff
@@ -0,0 +1,21 @@
+import { install } from "@twind/core"
+import config from "./twind.config"
+
+// Override `warning` event from Twind to suppress false warnings.
+// This override need to run before installing twind (`install(config)`).
+addEventListener('warning', (event) => {
+  event.preventDefault()
+
+  const warning = event.detail
+
+  // Skip warnings for css classes from Storybook
+  if (warning.code === "TWIND_INVALID_CLASS" && warning.detail.startsWith("sb-")) {
+    return
```

Comment:
These twind warnings will be suppressed:
![image](https://user-images.githubusercontent.com/7823011/206702071-6ebfb6d5-1ece-439d-941a-1ce28fcd8e9e.png)